docs/examples/droplet_fusion/droplet_fusion.rst

```diff
@@ -0,0 +1,200 @@
+
+Download the droplet fusion data
+--------------------------------
+
+The droplet fusion data are stored on zenodo.org.
+We can download the data directly from Zenodo using the function :func:`~lumicks.pylake.download_from_doi`.
+The data will be stored in the folder called `"test_data"`::
+
+    filenames = lk.download_from_doi("10.5281/zenodo.12772709", "test_data")
+
+Relaxation time of fusion event
+-------------------------------
+
+First, plot the force signal using Pylake. The force has a characteristic jump during a fusion event. 
+Since the force calibration breaks down during the fusion event, the absolute value of the force is not reliable, and we label the y-axis as 'laser signal'::
+
+    plt.figure()
+    f["Force HF"]["Force 2x"].plot()  
+    plt.ylabel("Laser signal (a.u.)")
+    plt.show()
+
+.. image:: force_signal.png
+
+Select data for fit
+^^^^^^^^^^^^^^^^^^^
+Below we are selecting the force and trap data at the fusion event. When fitting the fusion relaxation time, 
+it is important that the traps holding the droplets are either both static, or one of the traps is moving at a constant speed. 
+Therefore, we plot the trap position over time::
+
+    start = "5.4s"
+    stop = "5.9s"
+
+    force_selection = f.force2x[start:stop]
+    trap_selection = f["Trap position"]["1X"][start:stop]
+
+    plt.figure()
+    plt.subplot(2, 1, 1)
+    force_selection.plot()
+    plt.ylabel("Laser signal (a.u.)")
+    plt.subplot(2, 1, 2)
+    trap_selection.plot()
+    plt.ylabel("x-coordinate ($\mu$m)")
+    plt.tight_layout()
+    plt.show()
+
+.. image:: selected_data.png
+
+Model for fusion
+^^^^^^^^^^^^^^^^
+
+We will be fitting the force data during the fusion event with the following equation: $f(t) = ae^{-t/\tau}+bt+c$                 
+
+The term $bt$ accounts for the movement of the trap, assuming a constant trap speed [1]_. The parameter of interest is $\tau$, the relaxation time schale of the fusion event::
+
+    from scipy.optimize import curve_fit
+
+    def relaxation_model(t, tau, a,b,c):
+        return a*np.exp(-t / tau)+b*t+c
+
+Fit the data and plot the result::
+
+    time = force_selection.seconds
+    force = force_selection.data
+
+    popt, pcov = curve_fit(relaxation_model, time, force, [0.1,force[0],0,0])
+    print(popt)
+    plt.figure()
+    plt.plot(time,force)
+    plt.plot(time,relaxation_model(time,*popt),label=f'$\\tau$ = {popt[0]:0.2f}s')
+    plt.legend()
+    plt.show()
+
+.. image:: fit.png
+
+The relaxation time obtained from the fit is 0.05 seconds. 
+
+Now we will proceed to determined the size of the droplets before the fusion event.
+
+Droplet size
+------------
+
+First load the scan and print the relevant metadata::
+
+    >>> for name, scan in f.scans.items():
+            print(f"num frames: {scan.num_frames}")
+            frame_duration = (scan.frame_timestamp_ranges()[0][1]-scan.frame_timestamp_ranges()[0][0])/1e9
+            print(f"frame duration: {frame_duration} s")
```

Comment:
This will not compile to a jupyter notebook correctly. If you want a multi-line command with output, you have to extend the input lines with `...` (e.g.)
```suggestion
    >>> for name, scan in f.scans.items():
    ...     print(f"num frames: {scan.num_frames}")
    ...     frame_duration = (scan.frame_timestamp_ranges()[0][1]-scan.frame_timestamp_ranges()[0][0])/1e9
    ...     print(f"frame duration: {frame_duration} s")
```